### PR TITLE
chore: remove unused imports

### PR DESCRIPTION
**Pull request by DaniPopes**

### CODE REVIEW

**Review by Evalir — APPROVED** [review]

File: crates/forge/bin/cmd/script/transaction.rs [hunk 1]
```diff
@@ -410,7 +385,7 @@ pub mod wrapper {
         #[serde(rename = "gasUsed")]
         pub gas_used: Option<U256>,
         /// Contract address created, or `None` if not a deployment.
-        #[serde(rename = "contractAddress", serialize_with = "serialize_opt_addr")]
```
Comment (onbjerg) [inline crates/forge/bin/cmd/script/transaction.rs:413]: do we not need this?

Comment (DaniPopes) [reply crates/forge/bin/cmd/script/transaction.rs:413]: serialize fns don't do anything different than default impl

Comment (onbjerg) [reply crates/forge/bin/cmd/script/transaction.rs:413]: default impl does checksum? if so, all good, think this was just leftover from ethers

Comment (DaniPopes) [reply crates/forge/bin/cmd/script/transaction.rs:413]: Ah you're right, we wrapped to checksum, I don't know why but I'll keep it

**Review by onbjerg — APPROVED** [review]

**Review by mattsse — APPROVED** [review]

**Pull request merged**